app/grants/templates/grants/cancel.html

```diff
@@ -14,12 +14,13 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
-{% load static humanize i18n %}
+{% load static humanize i18n grants_extra %}
```

Comment:
What is grants_extra doing? I haven't seen this yet.

Reply:
@captnseagraves https://github.com/gitcoinco/web/pull/2946#discussion_r236353688 
I would appreciate some help making this nicer 😅

Reply:
`grants_extra` has modulo helper function https://github.com/gitcoinco/web/pull/2946#issuecomment-441732758

Reply:
@thelostone-mc could we put this logic in the `grants/views.py` `grant_new` function at ln160? Pseudocode: `if logo == None perform logic at ln65 of grants/cancel`.  This address this issue on object creation and puts the logic in a better place than the template.